cordova/build.json

```diff
@@ -4,7 +4,8 @@
             "keystore": "release.keystore",
             "storePassword": "betaflight",
             "alias": "betaflight",
```

Comment:
Can we change this to "rotorflight"... ?

Reply:
Created a new key.